src/index.html

```diff
@@ -4,11 +4,52 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home-page">
+    <header class="header">
+        
+      <a class="logo" href="#home-page">
+        <img src="./images/logo.png" alt="Logo Moyo">
+      </a>
+      
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
   </body>
```

Comment:
```suggestion
    </header>
  </body>
```
empty line only for same nesting-level elements